src/rajaperf_config.hpp.in

```diff
@@ -23,6 +23,7 @@
 #include "RAJA/config.hpp"
 #include "camp/number.hpp"
 
+#include <algorithm>
```

Comment:
Move this under line 42 since it's only used for caliper?

Reply:
Yeah that would be cleaner

Reply:
Can you move it? I don't think I can modify your branch.

Reply:
> Can you move it? I don't think I can modify your branch.

Ok, done.